api/download.py

```diff
@@ -125,33 +151,6 @@ def _find_new_path(path, list_used_subpaths):
         used_subpaths[parent_id] = used_subpaths.get(parent_id, []) + [path]
         return path
 
-    def _archivestream(self, ticket):
-        BLOCKSIZE = 512
-        CHUNKSIZE = 2**20  # stream files in 1MB chunks
-        stream = cStringIO.StringIO()
-        with tarfile.open(mode='w|', fileobj=stream) as archive:
-            for filepath, arcpath, _ in ticket['target']:
-                yield archive.gettarinfo(filepath, arcpath).tobuf()
-                with open(filepath, 'rb') as fd:
-                    for chunk in iter(lambda: fd.read(CHUNKSIZE), ''):
-                        yield chunk
-                    if len(chunk) % BLOCKSIZE != 0:
-                        yield (BLOCKSIZE - (len(chunk) % BLOCKSIZE)) * b'\0'
-        yield stream.getvalue() # get tar stream trailer
-        stream.close()
-
-    def _symlinkarchivestream(self, ticket, data_path):
-        for filepath, arcpath, _ in ticket['target']:
-            t = tarfile.TarInfo(name=arcpath)
-            t.type = tarfile.SYMTYPE
-            t.linkname = os.path.relpath(filepath, data_path)
-            yield t.tobuf()
-        stream = cStringIO.StringIO()
-        with tarfile.open(mode='w|', fileobj=stream) as archive:
-            pass
-        yield stream.getvalue() # get tar stream trailer
-        stream.close()
-
```

Comment:
Was this change required as part of this PR?

Am I reading it correctly that the only change is a slight rename, and moving out of the `Download` class?